packages/@aws-cdk/aws-ecr-assets/lib/image-asset.ts

```diff
@@ -5,7 +5,7 @@ import * as fs from 'fs';
 import * as path from 'path';
 import { AdoptedRepository } from './adopted-repository';
 
-export interface DockerImageAssetProps extends assets.CopyOptions {
+export interface DockerImageAssetProps extends assets.FingerprintOptions {
```

Comment:
This is a bit confusing. Why make it extend `FingerprintOptions`? This means the `FingerprintOptions.extra` property is now exposed to users, intentional?

Actually, I guess the same argument can be made for extending `CopyOptions` in the first place. Seems like this type if information is not the user's concern.

Reply:
This is intentionally exposed to users, so they have control over how the asset is used. For example, they can decide to exclude certain files, or add their own salt to the fingerprint.

Reply:
Got it 👍